src/core/Akka/Actor/ActorCell.DefaultMessages.cs

```diff
@@ -353,8 +417,8 @@ private void Create(Exception failure)
         /// </summary>
         public virtual void Start()
         {
-            PreStart();
-            Mailbox.Start();
+            // This call is expected to start off the actor by scheduling its mailbox.
+            Dispatcher.Attach(this);
```

Comment:
`Dispatcher.Attach` starts the mailbox and registers the actor's mailbox for execution